src/interactUtils.js

```diff
@@ -74,18 +74,28 @@ interactHelper = {
             eventOptions = {
                 bubbles: true,
                 pointerId: 1,
-                bubbles: true, 
-                cancelable: true, 
-                pointerType: "touch",
+                cancelable: true,
+                pointerType: 'touch',
                 width: 100,
                 height: 100,
                 isPrimary: true
             };
         if (element) {
             domElement = element instanceof $ ? element.get(0) : element;
 
-            domElement.dispatchEvent(new PointerEvent('pointerdown', eventOptions));
-            domElement.dispatchEvent(new PointerEvent('pointerup', eventOptions));
+            var firstEvent;
+            var secondEvent;
```

Comment:
`var` variable declarations SHOULD be on the top of the scope.

Please either transform the whole thing into ES6, or move them upper.

Reply:
Done